resources/test-runner.mjs

```diff
@@ -0,0 +1,96 @@
+import { TEST_INVOKER_LOOKUP } from "./test-invoker.mjs";
+
+export class TestRunner {
+    constructor(suite, test, params, callback, page, frame) {
```

Comment:
Can we use a consistent parameter order as SuiteRunner? so "frame, page, params"

Reply:
changed order for both SuiteRunner and TestRunner